cadl-tests/cadl/response.cadl

```diff
@@ -46,8 +47,16 @@ interface ResponseOp {
     @header("operation-location")
     operationLocation: ResourceLocation<Resource>;
   };
+}
+
+// operation using Azure.Core
+interface CadlCoreOp {
```

Comment:
separate the testing of cadl-azure-core related request/response in another interface.